go/performance/benchmarks/benchmark_helper.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/doltdb"
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/env"
+	"github.com/liquidata-inc/dolt/go/libraries/utils/filesys"
+	"github.com/liquidata-inc/dolt/go/store/types"
+	"math/rand"
+	"strconv"
+	"strings"
+)
+
+const (
+	testHomeDir = "/user/tester"
+	workingDir  = "/user/tester/some/testDir"
+
+	// supported GenTypes
+	increment = GenType("increment")
+	random = GenType("random")
+)
+
+func testHomeDirFunc() (string, error) {
+	return testHomeDir, nil
+}
+
+func createTestEnvWithFS(fs filesys.Filesys) *env.DoltEnv {
+	const name = "test mcgibbins"
+	const email = "bigfakeytester@fake.horse"
+	dEnv := env.Load(context.Background(), testHomeDirFunc, fs, doltdb.InMemDoltDB)
+	err := dEnv.InitRepo(context.Background(), types.Format_7_18, name, email)
+	if err != nil {
+		panic("Failed to initialize environment")
+	}
+	return dEnv
+}
+
+// GenType specifies how to generate subsequent row values for a given SeedColumn, for a test dataset
+type GenType string
+
+// SeedColumn is used to create a column in a test dataset for benchmark testing
+type SeedColumn struct {
+	Name string
+	PrimaryKey bool
+	Type types.NomsKind
```

Comment:
Using this with the hope of supporting additional `noms` types for benchmark testing.